lexos/static/js/scripts_base.js

```diff
@@ -1,3 +1,23 @@
+$(function () {
+  if (localStorage.getItem('visited') !== 'yes') {
+    localStorage.setItem('visited', 'yes')
+    $('#toggler').popover({
+      'html': 'true',
+      'title': 'In the Margins!',
+      'content': 'View instructions for any screen in Lexos by clicking the In the Margins Tab!\n<div class="text-center"><button type="button" id="gotit" class="btn btn-primary" style="background-color: #0068AF">Got it!</button></div>'
+    }).popover('show').data('bs.popover').tip().css({
+      'width': '170px',
+      'text-align': 'center'
+    })
+    $('#gotit').on('click', function () {
```

Comment:
I don't really remember if we agreed on using ```.click``` or ```on('click')```..

Reply:
I believe we actually agreed on .click. Just changed it.